index.js

```diff
@@ -0,0 +1,106 @@
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+import Books from './modules/class/books.class.js';
+import selector from './modules/selectors/selector.js';
+
+let tab = 'book-list'; // tab
+
+const booksClass = new Books(); // calling books class
+
+const updateStorage = () => {
+  const stringfiedBL = JSON.stringify(booksClass.getBooks());
+  localStorage.setItem('booklist', stringfiedBL);
+};
+
+const renderBooks = () => {
+  const Books = booksClass.getBooks();
+  let li = '';
+  if (Books.length) {
+    Books.forEach((book, index) => {
+      li += `<li class="book-item ${index % 2 ? '' : 'bg-black'}">
+       <div class="book-title">"${book.title}" by ${book.author}</div>
+       <button class="btn-remove" id=${book.id}>remove</button>
+   </li>`;
+    });
+  } else {
+    li += '<li class="center">Book list is empty!</li>';
+  }
+
+  selector.bookContainer.innerHTML = li;
+  const removeBtn = document.querySelectorAll('.btn-remove');
+  removeBtn.forEach((button) => button.addEventListener('click', () => {
+    booksClass.removeBook(button.id);
+    updateStorage();
+    renderBooks();
+  }));
+};
+
+const addBooks = (title, author, id) => {
+  booksClass.addBooks(title, author, id);
+  renderBooks();
+  updateStorage();
+  selector.message.textContent = 'Book added';
+  selector.TitleInputField.value = '';
+  selector.AuthorInputField.value = '';
+};
+
+function updateUserInterface() {
+  const retrievedData = localStorage.getItem('booklist');
+  tab = localStorage.getItem('tab');
+  const bookList = JSON.parse(retrievedData);
+  bookList.forEach((book) => booksClass.addBooks(book.title, book.author, book.id));
+  renderBooks();
+}
```

Comment:
- [ ] Please, try to use arrow syntax for this function 👍

Reply:
Hello, The lintor will not pass this error. I  had not upload the node_modules folder.I did it because to avoid this  [eslint error](https://github.com/Dagic-zewdu/awesome-books-with-classes/runs/5181093656?check_suite_focus=true) . This module is also in the requirement to import it so please take this it to account

Reply:
This issue is well resolved! ✔️